### PR TITLE
Update UI, bind API

### PR DESCRIPTION
**Pull request by hanle9**
- Bind partner api 
- Fix crash in android app and setting screen
- Update layout send screen
- Fix some issue layout of some screen/modal

### CODE REVIEW

**Comment by hanle9** [comment]
@TaiTrien

File: app/components/Views/Partners/components/PartnerItem.js [hunk 1]
```diff
@@ -21,11 +23,23 @@ export default function PartnerItem({ imageSrc, onItemPress }) {
...
 	});
-
 	return (
 		<TouchableHighlight style={styles.wrapper} onPress={onItemPress} underlayColor={colors.grey}>
 			<View style={styles.imgWrapper}>
-				<Image source={imageSrc} resizeMode={'contain'} style={styles.img} />
+				<Image
+					source={{
+						uri:
+							'https://account.liquichain.io/meveo/api/rest/fileSystem/binaries/default/KlubCoinPartner/88540662-605e-4b2f-8028-3405b3d6b4ac/icon',
```
Comment (TaiTrien) [inline app/components/Views/Partners/components/PartnerItem.js:32]: We have the api for partner data already, please update data here

File: app/components/Views/FAQ/index.js [hunk 2]
```diff
@@ -34,6 +34,7 @@ class FAQScreen extends PureComponent {
 	async fetchQuestions() {
 		APIService.getFAQs((success, json) => {
 			this.questions = [...json];
+			console.log(json);
```
Comment (TaiTrien) [inline app/components/Views/FAQ/index.js:37]: remove log code

**Review by TaiTrien — CHANGES_REQUESTED** [review]
Amazing changes, please update new thing that I commented

File: app/components/Views/FAQ/index.js (hunk 2 above)
Comment (hanle9) [reply app/components/Views/FAQ/index.js]: I fixed it. Please review @TaiTrien

**Review by TaiTrien — APPROVED** [review]

**Pull request merged**